README.md

```diff
@@ -48,17 +48,17 @@ wizAssets.deleteFile("file://documents/settings/img/cards/card001.jpg", successC
 
 ### deleteFiles()
 
-**wizAssets.deleteFiles(Array manyURIs, Function success, Function fail );**
+**wizAssets.deleteFiles(Array localURIs, Function success, Function fail );**
 
-- delete all URIs in Array like; [ "file://documents/settings/img/cards/card001.jpg", "file://documents/settings/img/cards/card002.jpg " .. ] <br />
-- if you do not specify a filename only dir, then all contents of dir will be deleted; file://documents/settings/img/cards <br />
+- delete all URIs in Array like; [ "img/cards/card001.jpg", "img/cards/card002.jpg " .. ] <br />
+- if you do not specify a filename only dir, then all contents of dir will be deleted; img/cards <br />
 - the array CAN contain one URI string
 
 ### getFileURI()
 
-**wizAssets.getFileURI(String filePathWithFilename, Function success, Function fail);**
+**wizAssets.getFileURI(String localURI, Function success, Function fail);**
 
-- A success returns URI string like file://documents/settings/img/cards/card001.jpg <br />
+- A success returns a file URI string like file://documents/settings/img/cards/card001.jpg <br />
```

Comment:
Basically, if a user do not cache the file URI it got back using ```downloadFile```, he might want to get it again at a later time. I would prefer to keep these two around for now.